script/check_changelogs.py

```diff
@@ -5,9 +5,7 @@
 import os
 import requests
 
-
-VERSIONS = ["6.0", "6.1", "6.2", "6.3", "6.4", "6.5", "6.6", "6.7", "6.8", "7.0", "7.1", "7.2", "7.3", "7.x"]
-
+SUPPORTED_VERSIONS = [ "6.8", "7.5", "7.6", "7.x"]
```

Comment:
Nice. This is much easier to update. I noticed we're including 6.8 now, which makes sense as it's supported. It's showing "failed" because 6.8.6 release notes weren't moved forward. I'll fix that!

Reply:
Opened https://github.com/elastic/apm-server/pull/3191 and backport PRs to fix this.